packages/consumption/src/modules/requests/itemProcessors/readAttribute/AcceptReadAttributeRequestItemParameters.ts

```diff
@@ -31,7 +31,7 @@ export class AcceptReadAttributeRequestItemParameters extends Serializable {
         return typeof this.existingAttributeId !== "undefined";
     }
 
-    public isWithNewAttribute(): this is { newAttributeValue: IdentityAttribute | RelationshipAttribute } {
+    public isWithNewAttribute(): this is { newAttribute: IdentityAttribute | RelationshipAttribute } {
```

Comment:
How is the _is with new attribute_ to be parsed? As in "contains a new attribute"?

Reply:
Accepting a ReadAttributeRequestItem, you either do so using the `AcceptReadAttributeRequestItemParametersWithExistingAttribute` or `AcceptReadAttributeRequestItemParametersWithNewAttribute`, depending on whether you want to share an existing Attribute (Repository/ThirdPartyRelationship) or you want to create a new one (own shared/peer shared pair). Hence, one of `existingAttributeId` or `newAttribute` of `AcceptReadAttributeRequestItemParameters` must be set. The method `isWithNewAttribute()` should return true, if `newAttribute` is set.